src/openrct2/windows/footpath_scenery.c

```diff
@@ -0,0 +1,484 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
@ccoors Yes, the C code doesn't have a lot of common style, but we try to keep things sane. Rule of a thumb is: try doing what surrounding code does. C++ code *mostly* has a proper style, which should be close to what `clang-format` does, but even there are discrepancies. We even [have that documented!](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-style)

@marijnvdwerf I think the date is required. https://www.gnu.org/licenses/gpl-3.0.en.html#howto lists it.